graphql/tests/query.rs

```diff
@@ -1755,3 +1755,43 @@ fn can_query_root_typename() {
         assert_eq!(extract_data!(result), Some(exp));
     })
 }
+
+#[test]
+fn deterministic_error() {
+    use serde_json::json;
+    use test_store::block_store::BLOCK_TWO;
+
+    run_test_sequentially(|store| async move {
+        let deployment = setup(
+            store.as_ref(),
+            "testDeterministicError",
+            BTreeSet::new(),
+            IdType::String,
+        )
+        .await;
+
```

Comment:
Not necessary for your PR, but I would like for our tests to populate `latest_ethereum_block_{number,hash}` before failures, like this:

```rust
        test_store::transact_and_wait(
            &STORE.subgraph_store(),
            &deployment,
            BLOCKS[0].clone(),
            vec[]!,
        )
        .await
        .unwrap();
```

Reply:
Yhea these tests are a bit strange in their setup but I'll leave that for another time.